libappstream-glib/as-store.c

```diff
@@ -2788,7 +2788,7 @@ as_store_search_per_user (AsStore *store,
 						 "flatpak",
 						 "exports",
 						 "share",
-						 "appdata",
+						 "metainfo",
```

Comment:
So this code path isn't going to be used for the next gnome-software release onwards, so i think we can ignore this for now.